custom_model_runner/datarobot_drum/drum/language_predictors/python_predictor/python_predictor.py

```diff
@@ -44,7 +44,13 @@ def mlpiper_configure(self, params):
 
         sys.path.append(self._code_dir)
         self._model_adapter.load_custom_hooks()
-        self._model = self._model_adapter.load_model_from_artifact()
+
+        try:
+            self._model = self._model_adapter.load_model_from_artifact()
+        except Exception as e:
```

Comment:
https://docs.python.org/3/tutorial/errors.html#exception-chaining

> Exception chaining happens automatically when an exception is raised inside an [except](https://docs.python.org/3/reference/compound_stmts.html#except) or [finally](https://docs.python.org/3/reference/compound_stmts.html#finally) section. This can be disabled by using from None idiom:

Reply:
remove the `from`